From 7d3e8ae1436f7b546c6b44b846324dba604007fd Mon Sep 17 00:00:00 2001
From: Yao Zhao <ca.yao.zhao@gmail.com>
Date: Wed, 10 Mar 2021 14:23:03 +1100
Subject: [PATCH] [xray] Fix xray document spelling

fix a couple of words spelling

Reviewed By: dberris

Differential Revision: https://reviews.llvm.org/D96658
---
 llvm/docs/XRay.rst        | 2 +-
 llvm/docs/XRayExample.rst | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/llvm/docs/XRay.rst b/llvm/docs/XRay.rst
index 72768fa8410a..6453ff7651be 100644
--- a/llvm/docs/XRay.rst
+++ b/llvm/docs/XRay.rst
@@ -47,7 +47,7 @@ You can use XRay in a couple of ways:
 - Instrumenting your C/C++/Objective-C/Objective-C++ application.
 - Generating LLVM IR with the correct function attributes.
 
-The rest of this section covers these main ways and later on how to customise
+The rest of this section covers these main ways and later on how to customize
 what XRay does in an XRay-instrumented binary.
 
 Instrumenting your C/C++/Objective-C Application
diff --git a/llvm/docs/XRayExample.rst b/llvm/docs/XRayExample.rst
index 5acf90f95fb2..c0a2a7a917cf 100644
--- a/llvm/docs/XRayExample.rst
+++ b/llvm/docs/XRayExample.rst
@@ -319,7 +319,7 @@ We then build the above with XRay instrumentation:
   $ XRAY_OPTIONS="patch_premain=true xray_mode=xray-basic" ./sample
 
 We can then explore the graph rendering of the trace generated by this sample
-application. We assume you have the graphviz toosl available in your system,
+application. We assume you have the graphviz tools available in your system,
 including both ``unflatten`` and ``dot``. If you prefer rendering or exploring
 the graph using another tool, then that should be feasible as well. ``llvm-xray
 graph`` will create DOT format graphs which should be usable in most graph
-- 
2.39.3

